docs/how-to/h-deploy-gke.md

```diff
@@ -0,0 +1,106 @@
+# Deploy Charmed MySQL K8s on GKE
+
+Google Kubernetes Engine (GKE) - the most scalable and fully automated Kubernetes service from Google. To access GKE WEB interface, open https://console.cloud.google.com/compute/
+
+# Install GKE and Juju tooling
+Install juju and gcloud tool using SNAP:
+```shell
+> sudo snap install juju --classic
+> sudo snap install kubectl --classic
+> sudo snap install google-cloud-cli --classic
+```
+Login to Google Account
+```shell
+> gcloud auth login
+
+Go to the following link in your browser:
+
+    https://accounts.google.com/o/oauth2/...
+
+Enter authorization code: 4/0Absad3s...
+
+You are now logged in as [your_account@gmail.com].
+```
+
+Now you need to associate this installation with GCloud project, using "Project ID" from [resource-management](https://console.cloud.google.com/cloud-resource-manager):
+```shell
+> gcloud config set project <PROJECT_ID>
+
+Updated property [core/project].
+```
+
+As a last step, install the Debian package `google-cloud-sdk-gke-gcloud-auth-plugin` using [Google manual](https://cloud.google.com/sdk/docs/install#deb).
+
+# Create new GKE cluster
+The following command will start three [compute engines](https://cloud.google.com/compute/) on Google Cloud (imagine them as three physical servers in clouds) and deploy K8s cluster there.  To simplify the manual, the following command will use high-availability zone `europe-west1` and compute engine type `n1-standard-4` (which can be adopted for your needs if necessary):
+```shell
+gcloud container clusters create --zone europe-west1-c $USER-$RANDOM --cluster-version 1.25 --machine-type n1-standard-4 --preemptible --num-nodes=3 --no-enable-autoupgrade
```

Comment:
preemptible good for dev env, but not for production
```suggestion
gcloud container clusters create --zone europe-west1-c $USER-$RANDOM --cluster-version 1.25 --machine-type n1-standard-4 --num-nodes=3 --no-enable-autoupgrade
```

Reply:
Applied.